app/screens/debug-screen/debug-screen.tsx

```diff
@@ -34,120 +39,191 @@ export const DebugScreen: ScreenType = () => {
+              addDeviceToken(client)
+            }
           }}
-          disabled={!tempToken}
         />
+        {__DEV__ && (
+          <>
+            <Button
+              title="Reload"
+              containerStyle={styles.button}
+              onPress={() => DevSettings.reload()}
+            />
+            <Button
+              title="Crash test"
+              containerStyle={styles.button}
+              onPress={() => {
+                crashlytics().log("Testing crash")
+                crashlytics().crash()
+              }}
+            />
+          </>
+        )}
+        <View>
+          <Text style={styles.textHeader}>Environment Information</Text>
+          <Text selectable>Galoy Instance: {appConfig.galoyInstance.name}</Text>
+          <Text selectable>GQL_URL: {appConfig.galoyInstance.graphqlUri}</Text>
+          <Text selectable>GQL_WS_URL: {appConfig.galoyInstance.graphqlWsUri}</Text>
+          <Text selectable>
+            USD per 1 sat: {usdPerSat ? `$${usdPerSat}` : "No price data"}
+          </Text>
+          <Text>Token Present: {String(Boolean(hasToken))}</Text>
+          <Text>Hermes: {String(Boolean(usingHermes))}</Text>
+          <Text style={styles.textHeader}>Update Environment</Text>
+          <ButtonGroup
+            {...testProps("Galoy Instance Button")}
+            onPress={(index) => {
+              const nextGaloyInstance = galoyInstances[index]
+              if (nextGaloyInstance !== newGaloyInstance) {
+                setNewToken(
+                  nextGaloyInstance === appConfig.galoyInstance.name ? token : "",
+                )
+                setNewGaloyInstance(nextGaloyInstance)
+              }
+            }}
+            selectedIndex={galoyInstances.findIndex(
+              (value) => value === newGaloyInstance,
+            )}
+            buttons={galoyInstances}
+            containerStyle={styles.container}
+          />
+          <GaloyInput
+            {...testProps("Input access token")}
+            label="Access Token"
+            placeholder={"Access token"}
+            value={newToken}
+            secureTextEntry={true}
+            onChangeText={setNewToken}
+            selectTextOnFocus
+          />
+          <Button
+            {...testProps("Copy access token")}
+            title="Copy access token"
+            containerStyle={styles.button}
+            onPress={async () => {
+              Clipboard.setString(newToken)
+              Alert.alert("Token copied in clipboard.")
+            }}
+            disabled={!newToken}
+          />
+          {newGaloyInstance === "Custom" && (
+            <>
+              <GaloyInput
+                label="Graphql Uri"
+                placeholder={"Graphql Uri"}
+                value={newGraphqlUri}
+                onChangeText={setNewGraphqlUri}
+                selectTextOnFocus
+              />
+              <GaloyInput
+                label="Graphql Ws Uri"
```

Comment:
These inputs do spell checking and autocomplete which makes it awkward to enter a custom domain.  Again I realise this feature will not be used very much but just raising it.